plugins/rbac-backend/src/service/enforcer-delegate.ts

```diff
@@ -70,14 +60,11 @@ export class EnforcerDelegate {
     const trx = externalTrx ?? (await this.knex.transaction());
 
     try {
-      await this.policyMetadataStorage.createPolicyMetadata(
-        source,
-        policy,
-        trx,
-      );
-      const ok = await this.enforcer.addPolicy(...policy);
+      const ok = await this.enforcer.addPolicy(...policy, source);
```

Comment:
I have been thinking about this some more. Do we need to store the source information any more with the policies if we are just going to reference the role metadata for the source?